jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -773,7 +787,12 @@ IGN_GAZEBO_DEPENDENCIES="libignition-common3-dev  \\
                          libignition-math6-eigen3-dev \\
                          libbenchmark-dev"
 
-if [[ -n "${IGN_GAZEBO_MAJOR_VERSION}" && ${IGN_GAZEBO_MAJOR_VERSION} -ge 3 ]]; then
+if [[ -n "${IGN_GAZEBO_MAJOR_VERSION}" && ${IGN_GAZEBO_MAJOR_VERSION} -ge 4 ]]; then
+  IGN_GAZEBO_DEPENDENCIES="${IGN_GAZEBO_DEPENDENCIES} \\
+                        libignition-rendering3-dev \\
```

Comment:
I think you can remove rendering for now, it's going to build `master` from source

Reply:
7aa837f6c7ea51d05c54e725164ec12c3aa023e0